modules/afprog/afprog.c

```diff
@@ -358,18 +430,55 @@ afprogram_dd_init(LogPipe *s)
     }
   log_pipe_append(&self->super.super.super, (LogPipe *) self->writer);
 
-  return afprogram_dd_reopen(self);
+  return restore_successful ? TRUE : afprogram_dd_reopen(self);
+}
+
+static inline void
+afprogram_dd_store_reload_info(AFProgramDestDriver * const self, GlobalConfig * const cfg)
+{
```

Comment:
The [C++ const convention](https://en.wikipedia.org/wiki/Const_%28computer_programming%29#C.2B.2B_convention) is used here, so in this case the pointer is qualified as a `const`, not to the referenced type.
On the other hand, I would attach the asterisk to the type, but this is not common in the current codebase.